chainerx_cc/chainerx/routines/manipulation_test.cc

```diff
@@ -0,0 +1,1081 @@
+#include "chainerx/routines/manipulation.h"
```

Comment:
We no longer test routines in C++ tests.
Could you remove this file?

Reply:
Oh okay. Didn't notice that.